src/components/tool/guide/EresearchJob/EresearchJob.js

```diff
@@ -93,6 +93,30 @@ const getConfigGroups = (config, onConfigChange = () => {
                 selected: (config) => config?.nodes > 0,
             }
         },
+        "wallTime": () => {
+            return {
+                element: (key, selected) => <ConfigDuration
+                    key={key}
+                    title="Walltime"
+                    description="The estimated total time for the job."
+                    selected={selected}
+                    onChange={onChange("wallTime")}
+                    value={config?.wallTime}
+                    inputPropsHour={{
+                        min: 1,
+                        max: 167, /*One week - 1*/
```

Comment:
1 week would be 168 hours, also comment is strangely formatted

Reply:
What happens it the user is setting 168 hours and 59 minutes? Shall we still allow it?

Reply:
Increase max hours to 168 and removed the comment

Reply:
Yeah I think so.